terraform/install.sh

```diff
@@ -128,7 +128,6 @@ displaySuccessMessage() {
     log "********************************************************************************"
     log "Stackdriver Sandbox deployed successfully!"
     log ""
-    log "     Stackdriver Dashboard: https://app.google.stackdriver.com/accounts/create"
```

Comment:
So the link won't work but there is a link to Cloud Monitoring that user needs to attend in order to create a workspace.

Reply:
Ok, I'll replace with the Cloud Monitoring page